lgc/include/lgc/state/AbiMetadata.h

```diff
@@ -242,4 +280,20 @@ union PA_CL_CLIP_CNTL {
   unsigned int u32All;
 };
 
+// VGT_SHADER_STAGES_EN register (just the GFX10 wave32 enable bits)
+union VGT_SHADER_STAGES_EN {
+  struct {
+    unsigned int : 8;
+    unsigned int DYNAMIC_HS : 1;
+    unsigned int : 12;
+    unsigned int HS_W32_EN : 1;
+    unsigned int GS_W32_EN : 1;
+    unsigned int VS_W32_EN : 1;
+    unsigned int NGG_WAVE_ID_EN : 1;
```

Comment:
I think here we need the field PRIMGEN_EN for NGG, not NGG_WAVE_ID_EN.

Reply:
Again, I will remove the ones I am not using.